### PR TITLE
[ Xedra Evolved ]Undine and Salamander no longer sweat

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Bugfixes "XE Undine and Salamanders no longer sweat"

#### Purpose of change
Undine could regenerate from sweating and didn't lose thirst from it and salamanders could damage themselves by sweating.  This should solve both of those problems.

#### Describe the solution
Adds , { "value": "SWEAT_MULTIPLIER", "multiply": -1 }

#### Describe alternatives you've considered
None

### CODE REVIEW

File: data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json [hunk 1]
```diff
@@ -140,7 +140,15 @@
     "purifiable": false,
     "profession": true,
     "points": 2,
-    "enchantments": [ { "values": [ { "value": "WEAKNESS_TO_WATER", "add": -2 }, { "value": "THIRST", "multiply": -1 } ] } ],
+    "enchantments": [
+      {
+        "values": [
+          { "value": "WEAKNESS_TO_WATER", "add": -2 },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json:146]:
```suggestion
          { "value": "WEAKNESS_TO_WATER", "add": -0.03 },
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "category": [ "SALAMANDER" ],
     "prereqs": [ "SALAMANDER_SKIN_1" ],
     "changes_to": [ "SALAMANDER_SKIN_3" ],
-    "enchantments": [ { "values": [ { "value": "WEAKNESS_TO_WATER", "add": 1 } ] } ],
+    "enchantments": [ { "values": [ { "value": "WEAKNESS_TO_WATER", "add": 1 }, { "value": "SWEAT_MULTIPLIER", "multiply": -1 } ] } ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json:26]:
```suggestion
    "enchantments": [ { "values": [ { "value": "WEAKNESS_TO_WATER", "multiply": 0.016 }, { "value": "SWEAT_MULTIPLIER", "multiply": -1 } ] } ],
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json [hunk 3]
```diff
@@ -140,7 +140,15 @@
     "purifiable": false,
     "profession": true,
     "points": 2,
-    "enchantments": [ { "values": [ { "value": "WEAKNESS_TO_WATER", "add": -2 }, { "value": "THIRST", "multiply": -1 } ] } ],
+    "enchantments": [
+      {
+        "values": [
+          { "value": "WEAKNESS_TO_WATER", "add": -0.03 },
```
[the remark this answers is not in the document]
Comment (Maleclypse) [reply data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json:146]: Yeah it made sense thematically compared to the fire ones taking damage but you’re probably right. I wonder if there’s a good way to either limit it to x hp per day like 5-10 or would it make sense to keep it and drop another decimal place or two

[the remark this answers is not in the document]
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json:146]: 5-10 sound too little with current healing system, but something like 30 per day (so 0.000347) sounds like a solid reason to sleep in the water

Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json:146]:
```suggestion
          { "value": "WEAKNESS_TO_WATER", "add": -0.000347 },
```

[the remark this answers is not in the document]
Comment (Maleclypse) [reply data/mods/Xedra_Evolved/mutations/paraclesians/paraclesian_mutations.json]: I hope you know how much I appreciate you!  Thank you!

**Pull request merged**